### PR TITLE
docs: cards added

### PR DESCRIPTION
**Pull request by rahulyadav139**
Card component is added.

### CODE REVIEW

**Review by aiman-ansari — CHANGES_REQUESTED** [review]
Everything is good UI looks so good .
Remove extra spaces and in card with ribbon it would look great if you rotate ribbon more in upward direction. Instead of horizontal change it into cross direction.

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,296 @@
+.shadow {
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (varun13169) [inline components/card/card.css:2]: Parameterize the color

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+.card.ribbon::after {
+  position: absolute;
+  content: attr(ribbon-content);
+  width: 5rem;
+  padding: 0.3125rem;
+  font-size: 0.625rem;
+  line-height: 0.625rem;
+  text-align: center;
+  background: linear-gradient(45deg, var(--gradient-primary-color));
+  color: var(--white-color);
```
Comment (varun13169) [inline components/card/card.css:59]: Try to give the meaningful color name

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+  padding: 2.5rem;
+  text-align: center;
+  visibility: visible;
+}
+
+.card-with-dismiss .btn-dismiss {
+  outline: none;
+  border: none;
+  background: transparent;
+  color: var(--white-color);
```
Comment (varun13169) [inline components/card/card.css:182]: Same as above.

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+.product-detail h2.title {
+  font-size: 1rem;
+  font-weight: normal;
+  color: var(--grey-color);
+  margin: 0.325rem 0;
+}
+.product-detail .rating {
+  padding: 0.325rem;
+  font-size: 0.75rem;
+  border: 1px solid var(--grey-color);
```
Comment (varun13169) [inline components/card/card.css:248]: same as above.

**Review by varun13169 — CHANGES_REQUESTED** [review]

**Comment by ShrutiLakhani** [comment]
1. In the Card with Text (Out of Stock) , you can mention it as Card with Text Overlay. 
2. Ribbon would appear better on top left rather than right.

Just these minor observations, rest everything looks great to me.

File: components/card/card.css (hunk 1 above)
Comment (manojsarna) [inline components/card/card.css:2]: create variables for colors

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+/* overlay */
+
+.card .overlay {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 10;
+  top: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.5);
```
Comment (manojsarna) [inline components/card/card.css:137]: check this color

**Review by manojsarna — APPROVED** [review]
overall code seems good.

GTM

**Comment by rahulyadav139** [comment]
> Everything is good UI looks so good . Remove extra spaces and in card with ribbon it would look great if you rotate ribbon more in upward direction. Instead of horizontal change it into cross direction.

Thank you for the review. Extra spaces have been removed and for rotating ribbon in the upward direction I will need to restructure both pseudo-element (before and after) and it would be too overwhelming for me. So I am rolling with this design style.

File: components/card/card.css (hunk 1 above)
Comment (rahulyadav139) [reply components/card/card.css:2]: creating a variable for this colour.

File: components/card/card.css [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+  font-size: 1.5rem;
+}
+.product-detail h2.title {
+  font-size: 1rem;
+  font-weight: normal;
+  color: var(--grey-color);
+  margin: 0.325rem 0;
+}
+.product-detail .rating {
+  padding: 0.325rem;
```
Comment (tarunsankhla) [inline components/card/card.css:246]: Try not using so much decimal place it will create a confusion

File: palasio-design.css [hunk 7]
```diff
@@ -1,5 +1,6 @@
 @import url('./components/main-style.css');
 @import url('./components/theme.css');
+@import url('./components/util.css');
```
Comment (tarunsankhla) [inline palasio-design.css:3]: Looks great just check the alert icon in mobile view it was getting little oval shape
Rest everything is good

**Pull request merged**